hammerdb/hammerdb_scripts/postgres/run_pg_tpcc.sh

```
@@ -0,0 +1,117 @@
+    sleep 30
+    starttime=`date +%Y.%m.%d.%T`
+    echo "Run with  ${uc} users"
+    sed -i "s/^diset tpcc pg_count_ware.*/diset tpcc pg_count_ware ${whc}/" runtest_pg.tcl
+    sed -i "s/^vuset.*/vuset vu ${uc}/" runtest_pg.tcl
+    ./hammerdbcli auto runtest_pg.tcl > test_pg_${Testname}_${uc}.out 2>&1
+    endtime=`date +%Y.%m.%d.%T`
+    grep RESULT test_pg_${Testname}_${uc}.out
+    tpm=`grep TPM test_pg_${Testname}_${uc}.out | awk '{print $7}'`
+    echo "\"${starttime}\",\"${endtime}\",\"${hostnm}\",\"${krel}\",\"Postgres\",\"${pgver}\",\"${numcpu}\",\"${totmem}\",\"${storagetype}\",\"${uc}\",\"${tpm}\"" >> user-benchmark-result.csv
+    cp user-benchmark-result.csv $benchmark_results_dir
+    cp /etc/my.cnf $benchmark_results_dir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)